store_data.py

```diff
@@ -0,0 +1,353 @@
+                                   "Google Cloud SQL instance"))
+  parser.set_defaults(create_instance=True)
+
+  CreateMutuallyExclusiveArgument(parser, "create_db",
+                                  "turn on if you want to create the DB")
+  parser.set_defaults(create_db=True)
+
+  CreateMutuallyExclusiveArgument(parser, "delete_db",
+                                  "turn on if you want to delete the DB")
+  parser.set_defaults(delete_db=True)
+
+  args = parser.parse_args()
+  logfile = open(args.logfile, "ab")
+
+  if args.create_instance:
+    sh_utils.RunGCloudService(["instances", "create", args.instance_name,
+                               "--tier", args.tier], args.project,
+                              "sql", logfile)
+    print "Google Cloud SQL Instance {} is created.".format(args.instance_name)
+  else:
+    print "Instance creation is skipped due to --no-create_instance."
+
+  sh_utils.RunGCloudService(["users", "set-password", args.username, "%",
+                             "--instance", args.instance_name, "--password",
+                             args.password], project=args.project,
+                            service="sql", logfile=logfile)
+  print "Usernames and passwords are set."
+
+  auth_ip_command = sh_utils.GetGcloud(
+      ["instances", "patch", args.instance_name,
+       "--authorized-networks", args.ownip],
+      project=args.project, service="sql")
+  auth_ip_command = " ".join(auth_ip_command)
+  pexpect.run(auth_ip_command,
+              events={"Do you want to continue (Y/n)?": "Y\n"},
+              logfile=logfile, timeout=None)
+  print ("This machine is configured to use the Google"
+         " Cloud SQL Instance {}").format(args.instance_name)
+
+  hostname = GetInstanceIP(args.instance_name, args.project)
+
+  if args.create_db:
+    connection = MySQLdb.connect(host=hostname, user=args.username,
+                                 passwd=args.password)
+    print "Connection successful."
+    db_utils.ExecuteAndReturnResult(connection, "CREATE DATABASE {};".format(
+        args.database))
+    connection.select_db(args.database)
+    print "Created DB."
+  else:
+    connection = MySQLdb.connect(host=hostname, user=args.username,
+                                 passwd=args.password, db=args.database)
+    print "Connection to DB {} is successful.".format(args.database)
+
+  # table will only be created if not exists
+  CreateTable(connection, args.table_name)
+  with open(args.json_result, "r") as f:
+    InsertIntoTable(connection, args.table_name, f)
+
+    connection.commit()
+
+  if args.delete_db:
+    db_utils.ExecuteAndReturnResult(connection,
+                                    "DROP DATABASE {};".format(args.database))
+    print "Database deletion is successful."
+  else:
+    print "Database deletion is skipped due to --no-delete_db."
+
+  connection.close()
+  print "DB Connection closed."
+
+if __name__ == "__main__":
+  main()
```

Comment:
How does this tie in with top-level invocation of the benchmark? It would be useful to be able to say "Run h2load with revision XYZ (git hash) of Envoy 5 times and put the results in a database with distinct timestamps but related run IDs". Then, I can later go and build a query for this run ID and maybe some others, draw the graph.